compute_endpoint/globus_compute_endpoint/strategies/base.py

```diff
@@ -69,7 +69,7 @@ def start(self, interchange):
         """Actually start the strategy
         Parameters
         ----------
-        interchange: funcx.executors.high_throughput.interchange.Interchange
+        interchange: gce.executors.high_throughput.interchange.Interchange
```

Comment:
From a typing perspective, should this be `globus_compute_endpoint.executors...`?  (And, though not the fault of _this_ PR, seems silly we never actually put the typing information into the method signature!)

Reply:
It's just a comment so I shortened it to gce to avoid need for line breaks, as it should have been obvious.  But expanded it if you you think it necessary.